alexmenuet-alexmenuet-taulia/hw4/BookManager/Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org'
 
-
+# Devise Authentication
+gem "devise"
+# Roadie email HTML layout
+gem 'roadie', '~> 2.4'
```

Comment:
What made you go with roadie? Seems to work well for this purpose, just curious. :)